src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/DynamicConfigFactory.java

```diff
@@ -225,12 +223,12 @@ public final boolean isInitialized() {
         return initialized.get();
     }
     
-    public void registerDCFListener(DCFListener listener) {
-        listeners.add(listener);
+    public void registerDCFListener(Object listener) {
```

Comment:
Since we're removing DCFListener maybe a more generic name like registerListener?

Reply:
I think it would be OK to keep the original name. It signifies the event bus the listener subscribes to. Note that in the past it was also not necessary to have `DCF` in the `registerDCFListener()` name, as it was clear that the registration is done for `DCFListener`.